valtimo-dependencies/gradle/publishing.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015-2021 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pluginManager.withPlugin('maven-publish') {
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                pom {
+                    name = 'Valtimo dependencies module'
+                    description = 'Valtimo dependencies module contains all modules that are part of Valtimo'
```

Comment:
Maybe change this to something like:

Valtimo dependencies module containing all the mandatory modules for Valtimo. Also contains version constraints.